src/Razor/src/Microsoft.VisualStudio.RazorExtension/UnifiedSettings/razor.registration.json

```diff
@@ -0,0 +1,102 @@
+{
+  "properties": {
+    "textEditor.razor.advanced.formatOnType": {
+      "type": "boolean",
+      "default": true,
+      "title": "%Setting_FormattingOnTypeDisplayName%",
+      "description": "%Setting_FormattingOnTypeDescription%",
+      "migration": {
```

Comment:
This isn't the correct format for migrations.  If you wanted to use pass-through migration, you need a "pass" object inside the "migration" object.  See *.registration.json in the devdiv AzDo server for examples.

Reply:
I see, I think I need 

```json
"pass":{
  "input": {
    "store": "VsUserSettingsRegistry",
    "path": "Razor\\CommitElementsWithSpace"
  }
}
```

Reply:
Updated, lmk if it looks right to you.